### DISCUSSION TITLE
Add the ability to change default communication device

### DISCUSSION THREAD

**Discussion by Chaython** [File-New-Project/EarTrumpet#1510] [Ideas]
https://github.com/File-New-Project/EarTrumpet/issues/348
A fork was made with the functionality but it has not been maintained and has not been merged to main... It also doesn't seem to work...
Would be nice to have a setting to make switching devices switch that device to default for both...
Right now I have to right click, - legacy - playback devices - right click the device - set as default communication device

**Comment by Chaython** [comment]
As a temporary solution I wrote https://github.com/Chaython/useful-registry-keys/blob/main/Defaultplaybackdevice.ps1
and assigned it to a hotkey using AHK
If anyone here wishes to utilize this, you'd have to change it depending on how many devices you have.

**Reply by RuinTD** [reply]
You could also use [NirSoft's SoundVolumeCommandLine](https://www.nirsoft.net/utils/sound_volume_command_line.html). For example:
```
svcl /SwitchDefault "Elgato Wave:3\Device\Headphones\Render" "Realtek(R) Audio\Device\Speakers\Render" all
```
This command will toggle between my headphones and speaker for my specific audio setup.
You can also change the `all` at the end to `2` if you only want to change your Communication device.
